www/webapp/src/router/index.js

```diff
@@ -180,13 +180,9 @@ router.beforeEach((to, from, next) => {
   } else {
     if (store.state.authenticated) {
       // Log in state was present, but not needed for the current page
-      // User restored a previous session. If navigation to home, divert to home page for authorized users
       if (recovered && to.name == 'home') {
+        // User restored a previous session. If navigation to home, divert to home page for authorized users
         next({name: 'domains'})
-      } else {
-        // user navigated to a page that doesn't require auth
-        // to bias on the safe side we log out
-        logout()
```

Comment:
That's already the case: at the top right, the "logout" button is present (white background, yellow outline) instead of the "login" button (yellow background)